src/bindings/crypto/bindings/gate-vector-napi.unit-test.ts

```diff
@@ -0,0 +1,67 @@
+import { expect } from 'expect';
+import { createRequire } from 'node:module';
+import { bindingsNapi } from '../bindings-napi.js';
+import type { Field, Gate, Wire } from './kimchi-types.js';
+
+const require = createRequire(import.meta.url);
+
+function loadNative() {
+  const candidates = [
+    '../../compiled/_node_bindings/plonk_napi.node',
+    '../../compiled/node_bindings/plonk_napi.node',
+  ];
+  for (const path of candidates) {
+    try {
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      return require(path);
+    } catch (err) {
+      if ((err as any).code !== 'MODULE_NOT_FOUND') throw err;
+    }
+  }
+  throw new Error('plonk_napi.node not found in compiled bindings');
+}
+
+const native: any = loadNative();
```

Comment:
should be able to do `import native from '@o1js/native-${platform}-${arch}'` soon (maybe once the packager is released)

I'd like to merge this so I can fix it up with the native packager as well